examples/crowdfund.vy

```diff
@@ -1,5 +1,13 @@
 # Setup private variables (only callable from within the contract)
-funders: {sender: address, value: wei_value}[int128]
+
+struct Funder :
+  sender: Address
+  value: wei_value
+
+struct Address :
+  x: address
```

Comment:
Is this done just as an example? Single member Structs, especially named as basic types, don't make a lot of sense in practice.

Reply:
Yep, I was testing nested structs. Plan to move it to a test.